package.json

```diff
@@ -99,37 +111,27 @@
     "tslint-no-circular-imports": "0.5.0",
     "typescript": "2.8.4"
   },
+  "engines": {
+    "node": ">=8.0.0 ",
+    "npm": ">=5.0.0 ",
+    "yarn": ">=1.0.0 "
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "homepage-comment": [
+    "*Do not* change this setting if you wish run Kiali under different server root.",
+    "Instead update 'web_root' in Kaili config map in OpenShift console."
+  ],
   "sassIncludes": {
     "src": "src",
     "patternfly": "node_modules/patternfly/dist/sass",
     "patternflyReact": "node_modules/patternfly-react/dist/sass",
     "bootstrap": "node_modules/bootstrap-sass/assets/stylesheets",
     "fontAwesome": "node_modules/font-awesome-sass/assets/stylesheets"
   },
-  "description": "React UI for [Kiali](https://github.com/kiali/kiali).",
-  "main": "index.js",
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/kiali/kiali-ui.git"
-  },
-  "keywords": [
-    "istio service mesh",
-    "kiali",
-    "openshift",
-    "observability",
-    "monitoring"
-  ],
-  "author": "Red Hat",
-  "license": "Apache-2.0",
-  "homepage-comment": ["*Do not* change this setting if you wish run Kiali under different server root.",
-                       "Instead update 'web_root' in Kaili config map in OpenShift console."],
-  "homepage": "./",
-  "publishConfig": {
-    "access": "public"
-  },
-  "engines": {
-    "node": ">=8.0.0 ",
-    "npm": ">=5.0.0 ",
-    "yarn": ">=1.0.0 "
-  }
+  "script-comments": [
```

Comment:
This was intended to go above "script", if not there, is not worth to have it at all.

Reply:
@josejulio please retest, I made your changes.